science/proxmark3-iceman/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qt5 1.0
+
+name                proxmark3-iceman
+
+github.setup        RfidResearchGroup proxmark3 4.13441 v
+github.tarball_from archive
+checksums           sha256  49c4f1854b364aa7ea7083581351f867128e71ea783d0ecd71fc41bcf7f63584 \
+                    rmd160  8ef2d5205f422466ca9de5fbfdc409dcdf2162db \
+                    size    24563979
+
+revision            0
+license             GPL-2
+categories          science comms
+platforms           darwin
+
+description         RRG/Iceman version of proxmark3 RFID Instrument Client/FW
+
+long_description    ${description}. By default, the repository is built for PM3RDV4. \
+                    To build for older hardware releases, please install the package \
+                    with the \"pm3generic\" variant.
+
+maintainers         {@linuxgemini linuxgemini.space:proxmark3packaging} openmaintainer
+
+depends_lib-append  port:readline \
+                    port:jansson \
+                    port:lua52 \
+                    port:python39
```

Comment:
please explicitly add `port:bzip2` as the resulting binary links with it. I realize that this is a library dependency of `python39`, so it will always be present; but being explicit here is better